recipe/meta.yaml

```diff
@@ -9,18 +9,24 @@ source:
   sha256: 68116cc184ab660a78a4cef323491e89909e5643b59c7b5f0a14f7c2b20e0a29
 
 build:
-  number: 0
+  number: 1
   skip: true  # [win]
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ stdlib("c") }}
     - {{ compiler('cxx') }}
+    - autoconf
+    - automake
+    - gettext
+    - m4
     - make
     - pkg-config
-
+    - xz-tools
+    - sed
   host:
+    - libtool
```

Comment:
Will libtool be required outside of the build environment?

Reply:
When cross-compiling, `libtool` needs to be in host otherwise it will use the wrong tool chain.  It doesn't have a run export since it's just an executable though.